docs/identity.md

```diff
@@ -1,10 +1,35 @@
 `dvc identity`
 ==============
 
-Update your DevCycle Identity.
```

Comment:
I'd put the "View or update" here as well

Reply:
Updated 👀 for reference this part is defined in the package.json instead: https://github.com/DevCycleHQ/cli/pull/270/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R114-R116